data/unk_8012ABC0.data.s

```diff
@@ -11,7 +11,7 @@
 
 # Unused
 glabel D_8012ABC0
```

Comment:
I'd love to know what this array was meant to be.

Reply:
there's also `data/unk_80009410.data.s` (in boot) and yeah I'd love to know too

Reply:
If I had to guess, I would say it isn't an array, probably an struct. But who knows? :s